packages/babel-traverse/src/scope/index.js

```diff
@@ -773,50 +803,24 @@ export default class Scope {
     this.uids = Object.create(null);
     this.data = Object.create(null);
 
-    // ForStatement - left, init
-
-    if (path.isLoop()) {
-      for (const key of (t.FOR_INIT_KEYS: Array<string>)) {
-        const node = path.get(key);
-        if (node.isBlockScoped()) this.registerBinding(node.node.kind, node);
-      }
-    }
-
-    // FunctionExpression - id
-
-    if (path.isFunctionExpression() && path.has("id")) {
-      if (!path.get("id").node[t.NOT_LOCAL_BINDING]) {
+    // TODO: explore removing this as it should be covered by collectorVisitor
```

Comment:
attempt to remove this has resulted in 14~ failing tests, most of the updated snapshots seemed OK, but some didn't. It requires more investigation and I'm afraid that I won't have time to do this

Reply:
:+1: